app/components/auk/abbreviated-text.hbs

```diff
@@ -0,0 +1,23 @@
+{{#if this.textTooLong}}
+  <div class="auk-o-flex">
+    {{#if (has-block)}}
+      {{yield this.abbreviatedText}}
+    {{else}}
+      {{this.abbreviatedText}}
+    {{/if}}
+    <WebComponents::AuButtonLink
+      @icon="circle-more"
+      @skin="mute"
+      @layout="icon-only"
+      class="auk-o-flex--vertical-center"
+    />
+    {{!-- TODO: This tooltip should be within the button-link, but doesn't yield when inside --}}
+    <EmberTooltip @tooltipClass="auk-tooltip">{{@text}}</EmberTooltip>
+  </div>
+{{else}}
+  {{#if this.isCustom}}
+    {{yield @text}}
+  {{else}}
+    {{@text}}
+  {{/if}}
```

Comment:
Deze ook niet nodig (niet meer gebruikt) en kan opgelost worden met die `<Auk::AbbreviatedText ....... as |abbreviatedText|>` toch?

Reply:
this.isCustom werkt sowieso niet meer

Reply:
deze moet dan ook has-block zijn